collections/E10a3_agricultural_production_area_change/E10a3_agricultural_production_area_change.md

```diff
@@ -1,8 +1,9 @@
-## Agricultural production - area change
+## Agricultural production - active area percentage
 
-Earth Observation can provide timely information for monitoring border closures & transport restrictions and changing food demand on agricultural activities related to food production. 
-Satellite observations, such as from Sentinel-2, allow to map the cultivated fields and ongoing activities continuously throughout the harvesting period. 
 
-**Asparagus in Germany**
-Harvesting asparagus is a highly labour-intensive activity and involves seasonal workers who typically arrive from Bulgaria and Romania to support the German farmers during the harvest period from March to June.
-Sentinel-2 data allow an overview of the cultivated white asparagus fields and related activities.
+This indicator illustrates the ratio:
+
+\frac{Area_2019-Area_2020}{Area_2019}
```

Comment:
@aapopescu Sorry, like this no. Although this is the way to do it in rich markdown that GitHub uses but for the StacInfo, we are working with the [commonmark](https://commonmark.org/help/), but we additionally allow some custom HTML, so I have tried following html way `<sup>Area_2019-Area_2020</sup>&frasl;<sub>Area_2019</sub>` and it seems to work but I am not sure if would be visually acceptable for you:
![image](https://github.com/eurodatacube/eodash-assets/assets/19269667/1b7360fe-1fc4-479f-bcd4-9db99249b04d)

Reply:
Changed and merging